psi4/driver/p4util/procutil.py

```diff
@@ -280,19 +280,14 @@ def extract_sowreap_from_output(sowout, quantity, sownum, linkage, allvital=Fals
         if allvital:
             raise ValidationError('Aborting upon output file \'%s.out\' not found.\n' % (sowout))
         else:
-            ValidationError('Aborting upon output file \'%s.out\' not found.\n' % (sowout))
             return 0.0
     else:
         while True:
             line = freagent.readline()
             if not line:
                 if E == 0.0:
-                    if allvital:
-                        raise ValidationError(
-                            'Aborting upon output file \'%s.out\' has no %s RESULT line.\n' % (sowout, quantity))
-                    else:
-                        ValidationError(
-                            'Aborting upon output file \'%s.out\' has no %s RESULT line.\n' % (sowout, quantity))
+                    raise ValidationError(
+                        'Aborting upon output file \'%s.out\' has no %s RESULT line.\n' % (sowout, quantity))
```

Comment:
Not really a problem with what you've done in this PR, but I think we should try to settle on a standard for `"xxx"%(yyy)` vs `"xxx".format(yyy)`.  I think it's fair to say that the latter is becoming more canonical, and I'd prefer that be used for the two instances in this PR where the `%` version is used.  The `.format` is present up above, I noticed.

Reply:
I welcome @loriab 's thoughts on that matter, but I suspect I already know that answer

Reply:
I might have been the last Python `%` holdout and have switched to format for the QCArchive ecosystem.

Reply:
I'm definitely in favor of f-string > `.format` > `%s` in general. Been gleefully moving to f-strings whenever it'll shorten the code. But these fns in particular are dead code for sow/reap that last worked in py2, so I was only willing to edit enough to silence lgtm.

Reply:
If this is dead code, there's no need to change anything then. Just wanted to bring it up